### PR TITLE
Add property for actual bounding and interaction sphere value

### LINKED ISSUE #1828: Show the actual value for the bounding sphere in the UI
When the bounding sphere (and interaction sphere) property of a SGN is set to -1, it is computed internally by each renderable. For debugging, it would be useful to see the actual value of the bounding sphere in the UI. 

I recently added an API function to ask OpenSpace for this information (`openspace.boundingSphere(sgnIdentifier)`) which could potentially be used. Alternatively, it would be possible to add a read-only property whose sole purpose is to access the actual bounding sphere of the node, but not change it

### PR DESCRIPTION
**Pull request by WeirdRubberDuck**
Closes #1828 

Have been wanting this for a while, and now when we talked about adding other read-only properties to display information, I thought it could be a good (and simple) approach for this issue

image

### CODE REVIEW

File: src/scene/scenegraphnode.cpp [hunk 1]
```diff
@@ -109,6 +109,26 @@ namespace {
...
+        openspace::properties::Property::Visibility::AdvancedUser
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo EvalInteractionSphereInfo = {
+        "EvaluatedInteractionSphere",
+        "Evaluated Interaction Sphere (Read Only)",
+        "This read-only property contains the evaluated value for the interaction sphere. "
+        "This is the actual value that is used internally within the software. If the "
+        "InteractionSphere property is set to -1, it is the computed value, otherwise it "
+        "should match the one set in the property",
```
Comment (alexanderbock) [inline src/scene/scenegraphnode.cpp:128]:
```suggestion
        "matches the one set in the property",
```

File: src/scene/scenegraphnode.cpp [hunk 2]
```diff
@@ -109,6 +109,26 @@ namespace {
         openspace::properties::Property::Visibility::AdvancedUser
     };
 
+    constexpr openspace::properties::Property::PropertyInfo EvalBoundingSphereInfo = {
+        "EvaluatedBoundingSphere",
+        "Evaluated Bounding Sphere (Read Only)",
```
[the remark this answers is not in the document]
Comment (WeirdRubberDuck) [reply src/scene/scenegraphnode.cpp:114]: Not really.  The better option would be to address this issue https://github.com/OpenSpace/OpenSpace/issues/2966 and make it more obvious which properties are read-only (currently I find it a bit difficult to see in the UI)

Comment (WeirdRubberDuck) [inline src/scene/scenegraphnode.cpp:114]:
```suggestion
        "Evaluated Bounding Sphere",
```

File: src/scene/scenegraphnode.cpp [hunk 3]
```diff
@@ -109,6 +109,26 @@ namespace {
...
+        "This read-only property contains the evaluated value for the bounding sphere. "
+        "This is the actual value that is used internally within the software. If the "
+        "BoundingSphere property is set to -1, it is the computed value, otherwise it "
+        "matches the one set in the property",
+        openspace::properties::Property::Visibility::AdvancedUser
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo EvalInteractionSphereInfo = {
+        "EvaluatedInteractionSphere",
+        "Evaluated Interaction Sphere (Read Only)",
```
Comment (WeirdRubberDuck) [inline src/scene/scenegraphnode.cpp:124]:
```suggestion
        "Evaluated Interaction Sphere",
```

**Pull request merged**